api/src/main/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirResourceProvider.java

```diff
@@ -82,6 +86,18 @@ public MethodOutcome createCondition(@ResourceParam Condition newCondition) {
 		return FhirProviderUtils.buildCreate(conditionService.saveCondition(Condition30_40.convertCondition(newCondition)));
 	}
 	
+	@Update
+	public MethodOutcome updateCondition(@IdParam IdType id, @ResourceParam Condition newCondition) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update resource");
+		}
+		
```

Comment:
There should be an actual update implementation here right?

Reply:
Actually i think this is should remain like this. Am digging deep into mockitoJUnit which seems to be breaking most of the testing code